src/mobile/apps/auction_support/stylesheets/index.styl

```diff
@@ -153,3 +136,40 @@ checkbox-size = 20px
   display block
   input
     padding 10px 0 11px 0
+
+.registration-form-section.accept-cos
```

Comment:
This feels like another thing we could consider abstracting out, especially if it's going to be copied again. But we can evaluate during the upcoming stories.

Reply:
I agree- there are some differences though (`red-color` vs `bad-red-color` comes to mind)

Reply:
That could probably be overridden in a specific case if all other styles are being inherited. But we can consolidate in a future PR!